interfaces/PA-App/src/app/personal-components/phone-number/phone-number.component.spec.ts

```diff
@@ -12,9 +15,16 @@ describe('PhoneNumberComponent', () => {
     TestBed.configureTestingModule({
       declarations: [PhoneNumberComponent],
       imports: [
-        TranslateModule.forRoot()
+        TranslateModule.forRoot(),
+        HttpClientTestingModule
       ],
       schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      providers: [
```

Comment:
These `providers` should represent the dependencies that are defined/required in the `PhoneNumberComponent`-constructor; So I think the tests fail because `ConversationService` or `PaDataService` isn't provided, at least...